library/L1_Peripheral/example/watchdog.cpp

```diff
@@ -0,0 +1,61 @@
+#include <cstdint>
+#include <iterator>
+
+#include "L0_Platform/lpc40xx/LPC40xx.h"
+#include "L1_Peripheral/interrupt.hpp"
+#include "L1_Peripheral/lpc40xx/gpio.hpp"
+#include "L1_Peripheral/lpc40xx/watchdog.hpp"
+#include "L3_Application/task.hpp"
+#include "utility/time.hpp"
+#include "utility/rtos.hpp"
+#include "utility/log.hpp"
+
+uint32_t timer_value;
+uint16_t timer_warning;
+
+sjsu::lpc40xx::Watchdog watchdog;
+
+uint8_t button3_port = 0;
+uint8_t button3_pin = 29;
+sjsu::lpc40xx::Gpio button3(button3_port, button3_pin);
+
+uint8_t led3_port = 1;
+uint8_t led3_pin = 18;
+sjsu::lpc40xx::Gpio led3(led3_port, led3_pin);
+
+void FeedTask(void * pv_paramater)
+{
+  while (true)
+  {
+    watchdog.FeedSequence();
+  }
+}
+
+void HighCPUTask(void * pv_paramater)
+{
+  while (true)
+  {
+     printf("High Task! \n");
+     sjsu::Delay(500ms);
```

Comment:
Put a note here as to why you are using `sjsu::Delay()` rather than vTaskDelay for the readers of the example. They need to know that you are doing this because you want to starve the tasks below you to cause the watchdog to be asserted if the delay is too long.